src/06_tuples.py

```diff
@@ -35,10 +35,13 @@ def dist(a, b):
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(example):
+    for i in example:
```

Comment:
style note: `i` typically indicates and index. in python using the `for/in` syntax assigns the value at the index to your variable. My preference would be something like `for num in example:`

Reply:
Fixed!